ap/config/analysis_st.py

```diff
@@ -314,7 +314,7 @@ def make_jme_filenames(jme_aux, sample_type, names, era=None):
         "category_ids", "deterministic_seed",
     },
     "MergeSelectionMasks": {
-        "LHEWeight.originalXWGTUP", "normalization_weight", "process_id", "category_ids",
+        "LHEWeight.originalXWGTUP", "normalization_weight", "process_id", "category_ids", "ht", "n_jet", "jet1_pt",
```

Comment:
Only thinking out loud ... but should we maybe add a parent column for variables that are only meant to be produced for the cutflows? E.g. the variables could be named `"cutflow.ht", "cutflow.n_jet", "cutflow.jet1_pt"`, and then we would only need to put "cutflow" into the set of variables to keep.

Reply:
Good idea, I'll try that out

Reply:
For compute intensive variables, that are already there, one could also add a simple alias which would not even copy the column :)